### DISCUSSION TITLE
How to print QRCode ?

### DISCUSSION THREAD

**Discussion by jnbenoit-oandb** [tr3v3r/react-native-esc-pos-printer#181] [Q&A] [answered]
Hello,
My question is simple : is it possible to print QR Code with your Library ? I can't find the right option in the "BarCodeType" Enum...

The only one who works is the `BARCODE_CODE128_AUTO` : 

`await printerInstance.addBarcode({ data: "01010101", type: BarcodeType.BARCODE_CODE128_AUTO });`

I'm using the EPSON TM-P20II as a printer.

Thank's for your answers

**Comment by tr3v3r** [comment, marked as answer]
My answer will be also simple. Yes😁

Check please example folder in this repo. Under SimplePrinr screen

**Comment by jnbenoit-oandb** [comment]
My bad,
I've just found the solution, i have to use the  `addSymbol()` instead of `addBarcode()`

`await printerInstance.addSymbol({ data: "hello", type: SymbolType.SYMBOL_QRCODE_MODEL_2, size: 10})`
